e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java

```diff
@@ -1175,6 +1175,30 @@ public PreferencesAO setPreference(String preference, String value, boolean eyeI
             return this;
         }
 
+        public String getPreference(String preference) {
+            searchPreference(preference);
+            String[] strings = context().$(byClassName("CodeMirror-code"))
+                    .findAll(byClassName("CodeMirror-line")).texts().toArray(new String[0]);
+            return String.join("\n", strings);
```

Comment:
What if `strings` are empty or contain just one value? We should check it before joining with `\n`